lib/litmus_paper/dependency/base.rb

```diff
@@ -0,0 +1,16 @@
+module LitmusPaper
+  module Dependency
+    class Base
+      attr_reader :available
+
+      def available?
+        return @available unless @available.nil?
```

Comment:
This looks like it's caching the result, which seems undesirable in basically all cases. Unless I'm reading this wrong?